frontend/src/tests/UserDomainSelect.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { it, describe, beforeEach, expect } from 'vitest';
+import { mount } from '@vue/test-utils';
+import { UserType } from 'fam-app-acsctl-api';
+import type { VueWrapper } from '@vue/test-utils/dist/vueWrapper';
+import type { DOMWrapper } from '@vue/test-utils/dist/domWrapper';
+import UserDomainSelect from '@/components/grantaccess/form/UserDomainSelect.vue';
+
+describe('UserDomainSelect', () => {
+    let wrapper: VueWrapper;
+    let emitChange: unknown[][] | undefined;
+
+    let becidRadioBtn: DOMWrapper<HTMLInputElement>;
+    let idirRadioBtn: DOMWrapper<HTMLInputElement>;
+
+    beforeEach(async () => {
+        wrapper = mount(UserDomainSelect);
+        becidRadioBtn = wrapper.find('#becidSelect');
```

Comment:
Hi Jota, I just notice this, its minor.
It should be `bceid` NOT b`ec`id. Could you refactor your test variables (becidRadioBtn) and ids (including the file UserDomainSelect.ts)?
Thanks.

Reply:
oh, I'll fix it